### PR TITLE
Userbook

### PR DESCRIPTION
**Pull request by whitx**
# Description
A documents to understand how to configure some basics flow with PacketFence

# Impacts
Documentation

# Delete branch after merge
NO

# NEWS file entries
## New Features
* Created a new documentation to help understand how to configure some flows for PacketFence

### CODE REVIEW

**Comment by dwlfrth** [comment]
Assign reviewers

**Comment by whitx** [comment]
@extrafu To merge it with the Admin Guide? or leave it likethat

**Comment by cgx** [comment]
Please update the dates in `PacketFence_User_Guide-docinfo.xml` and the outdated screenshots of the billing tiers. Thanks!

**Comment by dwlfrth** [comment]
@whitx hop

**Comment by extrafu** [comment]
Skip that for v7.3

File: docs/PacketFence_User_Guide.asciidoc [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+After finishing configure your PKI in PacketFence we will now do the provisioners.
+
+Provisioning
+^^^^^^^^^^^^
+
+Provisioners allow devices to automatically configure themselves to connect to the proper SSID (if applicable), use the proper authentication method (e.g. EAP-TLS) and trust the CA certificate and any certificate signed by it.
+
+Provisioners are configured in the PacketFence administration GUI under `Advanced Access Configuration->Provisioners`.
+
+Add a new provisioner for each of the device's calsses to be supported amongst Android, Apple Devices and Windows.
```
Comment (atran-inverse) [inline docs/PacketFence_User_Guide.asciidoc:156]: calsses -> classes

Comment (extrafu) [reply docs/PacketFence_User_Guide.asciidoc:156]: No need to correct typos on this PR. Let's just take what we want from it and scrap the rest.

**Comment by extrafu** [comment]
Obsolete.

**Pull request closed without merging**